contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -32,22 +32,21 @@ contract ValidatorShare is IValidatorShare {
       - add rewards to pool rewards
       - returns total active stake for validator
      */
-        uint256 stakePower;
-        uint256 valStake = stakeManager.validatorStake(validatorId);
-        stakePower = valStake.add(activeAmount); // validator + delegation stake power
-        uint256 _rewards = stakePower.mul(_reward).div(_totalStake);
+        uint256 validatorStake = stakeManager.validatorStake(validatorId);
+        uint256 combinedStakePower = validatorStake.add(activeAmount); // validator + delegation stake power
+        uint256 _rewards = combinedStakePower.mul(_reward).div(_stakePower);
 
-        uint256 _valRewards = valStake.mul(_rewards).div(stakePower);
+        uint256 _validatorRewards = validatorStake.mul(_rewards).div(combinedStakePower);
         // add validator commission from delegation rewards
         if (commissionRate > 0) {
-            _valRewards = _valRewards.add(
-                _rewards.sub(_valRewards).mul(commissionRate).div(100)
+            _validatorRewards = _validatorRewards.add(
+                _rewards.sub(_validatorRewards).mul(commissionRate).div(100)
             );
         }
-        _rewards = _rewards.sub(_valRewards);
-        validatorRewards = validatorRewards.add(_valRewards);
-        rewards = rewards.add(_rewards);
-        return stakePower;
+        uint256 delegatorsRewards = _rewards.sub(_validatorRewards);
```

Comment:
Declaring this above line 41 and using it in line 43 might help readability.

Reply:
yeah i have this code in other branch "improve-naming-stake-manager" did same there.